src/Microsoft.Data.Sqlite.Core/Utilities/BundleInitializer.cs

```
@@ -34,12 +35,12 @@ public static void Initialize()
                 var rc = sqlite3_win32_set_directory(
                     SQLITE_WIN32_DATA_DIRECTORY_TYPE,
                     ApplicationDataHelper.LocalFolderPath);
-                SqliteException.ThrowExceptionForRC(rc, db: null);
+                Debug.Assert(rc == SQLITE_OK);
```

Note: Changed to asserts since the wrong version of e_sqlite.dll gets deployed with the new net5.0-windows TFM, so the API returns SQLITE_ERROR. It's safe to ignore errors here, since the worst thing that can happen is relative paths are resolved to a directory without write permissions, and the error message they get later when trying to open the connection will be a lot more useful anyway.